### PR TITLE
Add a practical guide to CUE

### PR DESCRIPTION
**Pull request by Ereski**
Change-type: patch
Signed-off-by: Carol Schulze <carol@balena.io>

### CODE REVIEW

File: docs/cue.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+
+```
+$ cue export example.cue
+{
+    "x": 10,
+    "y": 11.5
+}
+```
+
+Refence cycles are permitted as long as the cycle can be broken off with a value. For example:
```
Comment (lurch) [inline docs/cue.md]: Refence -> Reference?

Comment (Ereski) [reply docs/cue.md]: oh yeah, prob more typos too still gotta do a spell check pass

**Comment by Ereski** [comment]
This repo is so old CI is failing on newer clippy lints :(

**Comment by Ereski** [comment]
closed in favour of #59

**Pull request closed without merging**